src/test/regress/expected/citus_split_shard_columnar_partitioned.out

```diff
@@ -0,0 +1,812 @@
+        eventdatetime     date,
+        PRIMARY KEY (measureid, eventdatetime))
+    PARTITION BY RANGE(eventdatetime);
+    CREATE TABLE colocated_partitioned_table_2020_01_01 PARTITION OF colocated_partitioned_table FOR VALUES FROM ('2020-01-01') TO ('2020-02-01');
+    SELECT create_distributed_table('colocated_partitioned_table', 'measureid');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+    CLUSTER colocated_partitioned_table_2020_01_01 USING colocated_partitioned_table_2020_01_01_pkey;
+    -- create reference tables
+    CREATE TABLE reference_table (measureid integer PRIMARY KEY);
+    SELECT create_reference_table('reference_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
+    SELECT shard.shardid, logicalrelid, shardminvalue, shardmaxvalue, nodename, nodeport
+    FROM pg_dist_shard AS shard
+    INNER JOIN pg_dist_placement placement ON shard.shardid = placement.shardid
+    INNER JOIN pg_dist_node       node     ON placement.groupid = node.groupid
+    INNER JOIN pg_catalog.pg_class cls     ON shard.logicalrelid = cls.oid
+    INNER JOIN pg_catalog.pg_namespace ns  ON cls.relnamespace = ns.oid
+    WHERE node.noderole = 'primary' AND ns.nspname = 'citus_split_test_schema_columnar_partitioned'
+    ORDER BY logicalrelid, shardminvalue::BIGINT;
+ shardid |              logicalrelid              | shardminvalue | shardmaxvalue | nodename  | nodeport
+---------------------------------------------------------------------
+ 8970000 | sensors                                | -2147483648   | 2147483647    | localhost |    57637
+ 8970004 | sensorscolumnar                        | -2147483648   | 2147483647    | localhost |    57637
+ 8970001 | sensors_old                            | -2147483648   | 2147483647    | localhost |    57637
+ 8970002 | sensors_2020_01_01                     | -2147483648   | 2147483647    | localhost |    57637
+ 8970003 | sensors_news                           | -2147483648   | 2147483647    | localhost |    57637
+ 8970005 | sensorscolumnar_old                    | -2147483648   | 2147483647    | localhost |    57637
+ 8970006 | sensorscolumnar_2020_01_01             | -2147483648   | 2147483647    | localhost |    57637
+ 8970007 | sensorscolumnar_news                   | -2147483648   | 2147483647    | localhost |    57637
+ 8970008 | colocated_dist_table                   | -2147483648   | 2147483647    | localhost |    57637
+ 8970009 | colocated_partitioned_table            | -2147483648   | 2147483647    | localhost |    57637
+ 8970010 | colocated_partitioned_table_2020_01_01 | -2147483648   | 2147483647    | localhost |    57637
+ 8970011 | reference_table                        |               |               | localhost |    57637
+ 8970011 | reference_table                        |               |               | localhost |    57638
+(13 rows)
```

Comment:
The order of the last two rows can change, and this creates a flaky test. I suggest you add an order on nodeport to make the results deterministic in these cases.